examples/Clients/Greeter/Program.cs

```diff
@@ -35,9 +35,24 @@ static async Task Main(string[] args)
             var channel = new Channel("localhost:50051", credentials);
             var client = new Greeter.GreeterClient(channel);
 
-            var reply = client.SayHelloAsync(new HelloRequest { Name = "GreeterClient" });
-            Console.WriteLine("Greeting: " + (await reply.ResponseAsync).Message);
```

Comment:
I didn't realize you could await AsyncUnaryCall directly.

Reply:
Yes, there's a GetAwaiter()  method  for that purpose.